bin/prettier.js

```diff
@@ -16,6 +16,14 @@ const argv = minimist(process.argv.slice(2), {
 const filenames = argv["_"];
 const write = argv["write"];
 const stdin = argv["stdin"];
+const options = {
```

Comment:
actually, can you bring back to the format() function, it doesn't need to be a global constant anymore